### ISSUE TITLE
disable form-group wrapper in horizontal form

### ISSUE THREAD

**Issue by DSalko** [bootstrap-ruby/rails-bootstrap-forms#206]
Hello

I wonder if it's possible to disable `form-group` wrapper for some fields. I'm building horizontal form and want to put two text fields in one line. I can change `label_col` and `control_col`, but that doesn't make any sense with wrapper form-group in place.

```
<div class='row'>
<%= form.text_field :start_date, control_col: "col-sm-3"  %>
<%= form.text_field :end_date, control_col: "col-sm-3", label_col: "col-sm-3" %>
</div>
```

Any suggestions? Thanks.

**Comment by datWav** [comment]
I think at this point this is not possible. The only idea I have is to skip the libary and do this on you own.

**Comment by ohru** [comment]
+1

**Comment by mattbrictson** [comment]
Is this the same feature request as #89? If so I will close this as a duplicate and we can move the discussion there.

**Comment by mattbrictson** [comment]
Closed as likely duplicate of #89
